src/rustup-cli/self_update.rs

```diff
@@ -190,7 +190,7 @@ tools, but otherwise, install the C++ build tools before proceeding.
 "#;
 
 static TOOLS: &'static [&'static str] =
-    &["rustc", "rustdoc", "cargo", "rust-lldb", "rust-gdb", "rls"];
+    &["rustc", "rustdoc", "cargo", "rust-lldb", "rust-gdb", "rls", "clippy-driver", "cargo-clippy"];
```

Comment:
So if I remember the discussion on discord correctly, we don't actually need `clippy-driver` here, right?